From: Bastian Germann <bage@debian.org>
Date: Wed, 10 Sep 2025 16:24:40 +0200
Subject: Upgrade pyo3 version

---
 src/_bcrypt/Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/_bcrypt/Cargo.toml b/src/_bcrypt/Cargo.toml
index bbf3457..1ed4668 100644
--- a/src/_bcrypt/Cargo.toml
+++ b/src/_bcrypt/Cargo.toml
@@ -6,7 +6,7 @@ edition = "2018"
 publish = false
 
 [dependencies]
-pyo3 = { version = "0.23.5", features = ["abi3"] }
+pyo3 = { version = ">=0.23.5,<0.27", features = ["abi3"] }
 bcrypt = "0.15"
 bcrypt-pbkdf = "0.10.0"
 base64 = "0.22.1"
